openstack/cronus/templates/cronus/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.cronus.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: cronus-ingress
+  annotations:
+    disco: "true"
+    #vice-president: "true" deprecated 30/9/20
+    kubernetes.io/tls-acme: "true"
+  labels:
+    app: cronus
+spec:
+  rules:
+    - host: cronus.{{ .Values.config.region }}.cloud.sap
+      http:
+        paths:
+          - backend:
+              serviceName: cronus-service
+              servicePort: {{ .Values.cronus.http }}
+            path: /
+  tls:
+    - hosts:
+      - cronus.{{ .Values.config.region }}.cloud.sap
+      secretName: tls-secret-cronus-{{ .Values.config.region }}-cloud-sap
```

Comment:
can we unify a secret name? e.g. use only `tls-secret-cronus` as a name

Reply:
```
    - hosts:
      - cronus.{{ .Values.config.region }}.cloud.sap
      secretName: tls-secret-cronus
```